phoneApp/js/app.js

```diff
@@ -1,183 +1,137 @@
-				        lastName: 'Петров',
-				        email: 'IvanPetrov@ec.ua'
-				      },
-				        {
-				        name: 'Сергей',
-				        lastName: 'Сергеев',
-				        email: 'SergeiSergeev@ec.ua'
-				      },
-				        {
-				        name: 'Иван',
-				        lastName: 'Иванов',
-				        email: 'IvanIvanov@ec.ua'
-				      },
-				        {
-				        name: 'Александр',
-				        lastName: 'Александров',
-				        email: 'AlexAlex@ec.ua'
-				      },
-				        {
-				        name: 'Алекс',
-				        lastName: 'Смирнов',
-				        email: 'AlexSmirnov@ec.ua'
-				      },
-				        {
-				        name: 'Сергей',
-				        lastName: 'Волков',
-				        email: 'VolkovSergey@ec.ua'
-				      },
-				        {
-				        name: 'Мария',
-				        lastName: 'Шарапова',
-				        email: 'MariyaSharapova@ec.ua'
-				      },
-				        {
-				        name: 'Александр',
-				        lastName: 'Винник',
-				        email: 'AlexVinnik@ec.ua'
-				      },
-				        {
-				        name: 'Дарий',
-				        lastName: 'Смирнов',
-				        email: 'DariySmirnov@ec.ua'
-				      },
-				        {
-				        name: 'Елена',
-				        lastName: 'Лещенко',
-				        email: 'ElenaLeshenko@ec.ua'
-				      },
-				        {
-				        name: 'Ольга',
-				        lastName: 'Новикова',
-				        email: 'OlgaNovikova@ec.ua'
-				      },
-				        {
-				        name: 'Наталья',
-				        lastName: 'Шемякина',
-				        email: 'ShemyakinaN@ec.ua'
-				      },
-				        {
-				        name: 'Анна',
-				        lastName: 'Донцова',
-				        email: 'AnnaDontsova@ec.ua'
-				      },
-				        {
-				        name: 'Влад',
-				        lastName: 'Яма',
-				        email: 'VladYama@ec.ua'
-				      },
-				        {
-				        name: 'Кира',
-				        lastName: 'Воробьева',
-				        email: 'Kira1990@ec.ua'
-				      },
-				        {
-				        name: 'Виктор',
-				        lastName: 'Кривенко',
-				        email: 'ViktorKriv@ec.ua'
-				      }
-				    ],
-			activePage: 'contacts'
+				{
+					name: "Иван",
+					lastName: "Петров",
+					email: "IvanPetrov@ec.ua"
+				},
+				{
+					name: "Сергей",
+					lastName: "Сергеев",
+					email: "SergeiSergeev@ec.ua"
+				},
+				{
+					name: "Иван",
+					lastName: "Иванов",
+					email: "IvanIvanov@ec.ua"
+				},
+				{
+					name: "Александр",
+					lastName: "Александров",
+					email: "AlexAlex@ec.ua"
+				},
+				{
+					name: "Алекс",
+					lastName: "Смирнов",
+					email: "AlexSmirnov@ec.ua"
+				},
+				{
+					name: "Сергей",
+					lastName: "Волков",
+					email: "VolkovSergey@ec.ua"
+				},
+				{
+					name: "Мария",
+					lastName: "Шарапова",
+					email: "MariyaSharapova@ec.ua"
+				},
+				{
+					name: "Александр",
+					lastName: "Винник",
+					email: "AlexVinnik@ec.ua"
+				},
+				{
+					name: "Дарий",
+					lastName: "Смирнов",
+					email: "DariySmirnov@ec.ua"
+				},
```

Comment:
I guess such mock data no need anymore because you download them from the server ?